yt_idefix/data_structures.py

```diff
@@ -742,16 +735,13 @@ def _parse_parameter_file(self):
         super()._parse_parameter_file()
         # from here self.geometry is assumed to be set
 
-        # some versions of Pluto define field names in lower case
-        # so we normalize to upper case to avoid duplicating data
-        # in BaseVtkFields.known_other_fields
-        normalize_varnames = self.dataset_type == "pluto-vtk"
-
         # parse the grid
         with open(self.filename, "rb") as fh:
             coords = vtk_io.read_grid_coordinates(fh, geometry=self.geometry)
             self._field_offset_index = vtk_io.read_field_offset_index(
-                fh, coords.array_shape, upper_case_varnames=normalize_varnames
+                fh,
+                coords.array_shape,
+                default_field_list=unzip(self._field_info_class.known_other_fields)[0],
```

Comment:
```suggestion
                default_field_list=list(unzip(self._field_info_class.known_other_fields)[0]),
```

Reply:
I guess a list comprehension can also be used here and is more readable, right?
```suggestion
                default_field_list=[f[0] for f in self._field_info_class.known_other_fields],
```

Reply:
that's a good point, and it's actually much more readable. Thanks !